### ISSUE TITLE
Wrong SuspiciousAssignmentsInspection

### ISSUE THREAD

**Issue by deMoneIrk** [kalessil/phpinspectionsea#1582]
| Subject        | Details                                                                       |
| :------------- | :---------------------------------------------------------------------------- |
| Plugin         | Php Inspections (EA Extended) 4.0.4.1 |
| Language level | PHP 7.2  |

Current behaviour
--
I have code like this:
```php
$list = [];
$i = 0;
foreach($row as $col) {
    $list[$i++] = new InsertText();

    if ($col->getTextStyle() !== null) {
        $list[$i++] = new UpdateTextStyle($col->getTextStyle());
    }

    $list[$i++] = new MoveCursor(2);
}
```

So I had an message from Php Inspections at line ```$list[$i++] = new MoveCursor(2);```:

> [EA] $requestList[$i++] is immediately overridden, perhaps it was intended to use 'else' here.

Of course, it's not overridden because of incrementation of $i.

Expected behaviour
--
There is shouldn't be that message.

Environment details
---
PhpStorm 2020.1.2
Build #PS-201.7846.90, built on June 3, 2020
Licensed to Tech-Director.ru / EVGENIY NEVEROV
Subscription is active until June 22, 2020
Runtime version: 11.0.7+10-b765.53 x86_64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
macOS 10.15.5
GC: ParNew, ConcurrentMarkSweep
Memory: 2014M
Cores: 4
Registry: run.processes.with.pty=TRUE
Non-Bundled Plugins: mobi.hsz.idea.gitignore, ru.adelf.idea.dotenv, com.kalessil.phpStorm.phpInspectionsEA, pro.opcode.bitrix

**Comment by kalessil** [comment]
Fixed!
